src/commands/force/source/deploy.ts

```diff
@@ -149,6 +154,8 @@ export class Deploy extends DeployCommand {
         manifest: this.flags.manifest && {
           manifestPath: this.getFlag<string>('manifest'),
           directoryPaths: this.getPackageDirs(),
+          destructiveChangesPre: this.getFlag<string>('predestructivechanges'),
```

Comment:
would you want to validate that the files exist before trying to do ComponentSetBuilder stuff, or does it handle that pretty well?